activerecord/CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-*   Add support for setting the filename of the schema or structure dump in the database config.
+* Add a new option `:update_only` to `upsert_all` to configure the list of columns to update in case of conflict.
+
+  Before, you could only customize the update SQL sentence via `:on_duplicate`. There is now a new option `:update_only` that lets you provide a list of columns to update in case of conflict:
+
+    ```ruby
+    Commodity.upsert_all(
+    [
+        { id: 2, name: "Copper", price: 4.84 },
+        { id: 4, name: "Gold", price: 1380.87 },
+        { id: 6, name: "Aluminium", price: 0.35 }
+    ],
+    on_duplicate: [:price] # Only prices will be updated
+    )
+    ```
+
+  *Jorge Manrubia*
+ 
+* Add support for setting the filename of the schema or structure dump in the database config.
```

Comment:
Looks like spacing got tweaked further back in the changelog. It should have `*   ` indentation throughout.

Reply:
Oh good catch. That's Rubymine markdown editor always messing up with Rails' readme files. Thanks!